packages/bundler/src/modules/BundleManager.ts

```diff
@@ -29,11 +31,19 @@ import { IBundleManager } from './IBundleManager'
 import { MempoolEntry } from './MempoolEntry'
 import { MempoolManager } from './MempoolManager'
 import { ReputationManager, ReputationStatus } from './ReputationManager'
+import { ChainConfig, Common, Hardfork, Mainnet } from '@ethereumjs/common'
+import { EOACode7702Transaction } from '@ethereumjs/tx'
+import { AuthorizationList, EOACode7702TxData } from '@ethereumjs/tx/src/types'
+import { PrefixedHexString } from '@ethereumjs/util'
+import { toRlpHex } from '@account-abstraction/utils/dist/src/interfaces/EIP7702Authorization'
 
 const debug = Debug('aa.exec.cron')
 
 const THROTTLED_ENTITY_BUNDLE_COUNT = 4
 
+const TX_TYPE_EIP_7702 = 4
```

Comment:
Should it be in config or something? We already know that this type means different things on different networks

Reply:
I don't think so, the EIP-7702 once merged into mainnet will be a part of the EVM and any chain that wants Tx 4 to mean anything else will have to switch. I don't know if there is any reasonable chain that will not be compatible, do you?